fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -1489,21 +1489,9 @@ public long getReplicaCount() {
         return replicaCount;
     }
 
-    public void checkStableAndNormal(String clusterName) throws DdlException {
+    public void checkNormalStateForAlter() throws DdlException {
         if (state != OlapTableState.NORMAL) {
-            throw new DdlException("Table[" + name + "]'s state is not NORMAL. "
-                    + "Do not allow doing materialized view");
-        }
-        // check if all tablets are healthy, and no tablet is in tablet scheduler
-        boolean isStable = isStable(Env.getCurrentSystemInfo(),
```

Comment:
why remove this logic which check all tablets healthy?

Reply:
actually checkTableStable function will check table stable, no need to double check here